draft-irtf-cfrg-rsa-blind-signatures.md

```diff
@@ -395,6 +395,9 @@ An alternative solution to this problem of message blindness is to give signers
 message being signed is well-structured. Depending on the application, zero knowledge proofs
 could be useful for this purpose. Defining such a proof is out of scope for this document.
 
+Verifiers should check that, in addition to the signature validity, the unblinded message is 
```

Comment:
What kind of structure would you think of verifying here? The threat model is that the client controls the message, so relevant checks should be done by the verifier decoding the message.